README.md.mustache

```diff
@@ -97,7 +97,7 @@ opam install {{ opam_name }}{{^ opam_name }}coq-{{ shortname }}{{/ opam_name }}
 To instead build and install manually, do:
 
 ``` shell
-git clone --recurse-submodules https://github.com/{{ organization }}/{{ shortname }}.git
+git clone {{# submodule }}--recurse-submodules{{/ submodule }} https://github.com/{{ organization }}/{{ shortname }}.git
```

Comment:
This way, it avoids showing two spaces when there are no submodules (and it should also avoid spurious diffs):

```suggestion
git clone {{# submodule }}--recurse-submodules {{/ submodule }}https://github.com/{{ organization }}/{{ shortname }}.git
```